src/api/authGroup/group.js

```diff
@@ -395,30 +399,15 @@ const agp = {
 						group: ag,
 						products: [],
 						productCodedIds: []
-					}
+					};
 					coreProducts.map((p) => {
 						core.products.push(p.id || p._id);
 						core.productCodedIds.push(p.codedId);
 					});
 				}
 			}
-		} else if(permissions?.sub_group === ag && permissions?.core?.group === ag) {
```

Comment:
Realizing that this is a matter of product access and roles, not specific permissions.